### ISSUE TITLE
Release for docker alpine aarch/arm64 needed

### ISSUE THREAD

**Issue by observingeagle** [nginx-proxy/docker-gen#318]
Would it be possible to release a package/binary for aarch64/arm64 architecture?
I realy apprechiate your work here and would love to use it on personal systems. As armhf and armel were or are widely spred the current architecture is amr64 /aarch64 and it is widely used (multi core SoCs and online services like for example scaleway).
So would it be too much effort to realease a version for this architecture?

Would be great. Thanks mate.

**Comment by tkw1536** [comment]
Duplicate of #304 ?

**Comment by jmformenti** [comment]
I built docker-gen for arm64 is available [here](https://github.com/jmformenti/rpi-docker-packs/blob/master/nextcloud/proxy/docker-gen-linux-arm64-0.7.4.tar.gz).

These are the steps to generate ``docker-gen`` for ``arm64`` on ``Raspberry Pi 4`` using ``pi`` user:

```
sudo apt install golang
mkdir -p /home/pi/go/src/github.com/jwilder
cd /home/pi/go/src/github.com/jwilder
git clone https://github.com/jwilder/docker-gen.git
cd docker-gen
go get github.com/robfig/glock
export PATH=$PATH:/home/pi/go/bin
glock install github.com/jwilder/docker-gen
make get-deps
make
```
output: docker-gen

**Comment by buchdag** [comment]
closed by #327
